swap_meet/clothing.py

```diff
@@ -1,2 +1,9 @@
-class Clothing:
-    pass
\ No newline at end of file
+from swap_meet.item import Item
+
+class Clothing(Item):
+    
+    def __init__(self, category = "Clothing", condition = 0):
+        super().__init__(category, condition)
```

Comment:
The category for `Clothing` must be the string "Clothing".  Making category an optional parameter here still makes it possible to pass in an alternative string for category, which could result in an instance of `Clothing` with a category that isn't "Clothing".  I recommend passing "Clothing" in as a string literal directly into the super constructor:
```suggestion
    def __init__(self, condition = 0):
        super().__init__("Clothing", condition)
```